scripts/ci.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -xe
+
+if [ ! -z "$1" ]
+then
+    report_location=$1
+else
+    report_location=`pwd`
+fi
+
+./gradlew --no-daemon --info clean
+./gradlew --no-daemon --info build countDebugDexMethods -PdisablePreDex
+if [ -n "${GCLOUD_SERVICE_KEY+1}" ]; then
```

Comment:
@Numan1617 could you pls add some comments here to explain a bit more off this script!?
kthxbye